Migrate - Swift 5.7 and update deployment target to iOS16

Kedar-27 opened the pull request:
Hi, I have update the code files and done necessary changes for the project to run on iOS 16

gokulrajanpillai commented:
@Kedar-27 Thanks for the update!
I would recommend not checking in the Pods folder. Was anything in the Pod module QRCodeReader modified after installing it?
Else we can update .gitignore to ignore the Pods folder.

Kedar-27 commented:
Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7

gokulrajanpillai commented:
> Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7

Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder.
Just wanted to know why was the Pods folder included in the PR?

Kedar-27 commented:
> > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> 
> Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?

It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore

Kedar-27 commented:
> > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > 
> > 
> > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> 
> It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore

Done.. Please check

gokulrajanpillai commented on .gitignore:
```diff
@@ -40,6 +41,9 @@ xcuserdata/
 Carthage/Build
 
+# Cocoapods
+Pods/
```
Awesome 🔥

gokulrajanpillai commented:
> > > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > > 
> > > 
> > > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> > 
> > 
> > It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore
> 
> Done.. Please check

Thanks a lot. Can you also make a commit removing the Pods folder? it is still part of the pull request

gokulrajanpillai reviewed (changes requested):
Once the Pods folder is removed, we can squash merge to remove it from the history (y)

Kedar-27 commented:
> > > > > Yes...i had updated the Pod from 8.0.3 to 10.0... v8.0.3 does not support Swift 5.7
> > > > 
> > > > 
> > > > Didnt mean the update on the Podfile to use the version 10.1.0, but on the actual project under Pods folder. Just wanted to know why was the Pods folder included in the PR?
> > > 
> > > 
> > > It wasn't included in gitIgnore that's y it got included in the commit. I will update gitIgnore
> > 
> > 
> > Done.. Please check
> 
> Thanks a lot. Can you also make a commit removing the Pods folder? it is still part of the pull request

Yes I have removed it now.

Kedar-27 commented:
Thanks for accepting.

gokulrajanpillai commented:
Thank you for your contribution! 😃
